integration-tests/api/__tests__/admin/draft-order/draft-order.js

```diff
@@ -59,15 +69,11 @@ describe("/admin/draft-orders", () => {
         ],
       }
 
-      const response = await api
-        .post("/admin/draft-orders", payload, {
-          headers: {
-            Authorization: "Bearer test_token",
-          },
-        })
-        .catch((err) => {
-          console.log(err)
-        })
+      const response = await api.post("/admin/draft-orders", payload, {
+        headers: {
+          Authorization: "Bearer test_token",
+        },
+      })
       expect(response.status).toEqual(200)
```

Comment:
suggestion: Should we check that the discount exists?

Reply:
No need, there's already multiple tests covering discounts in this test suite.

Reply:
when we add a discount during the creation of a draft order?

Reply:
https://github.com/medusajs/medusa/blob/33bb6044c800620ab97183680d8b9702b43f63d4/integration-tests/api/__tests__/admin/draft-order/draft-order.js#L466-L516

Reply:
so I don't understand something, how this test was passing while it was not working. we agree that it is an existing test

Reply:
It was passing since it is in a tax-exclusive context.